Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN apt-get update && \
 
 COPY requirements_all.txt requirements_all.txt
 RUN pip3 install --no-cache-dir -r requirements_all.txt
+RUN pip3 install --no-cache-dir colorlog
```

Comment:
Oh wow, I didn't realise that each block created a layer! That's excellent. I'll move it higher in the file.